recordium/main.py

```diff
@@ -51,7 +62,7 @@
 
 
 def debug_trace():
-    '''Set a tracepoint in the Python debugger that works with Qt'''
+    """Set a tracepoint in the Python debugger that works with Qt"""
```

Comment:
End the docstring with a "." (yes, I know this debug_trace already was there, but as you're touching the docstring...)